FreshRSS/FreshRSS
# Improved: Slider: Close area = grey background

When a slider is open, the left area is clickable and closes the slider.

After:
grey transparent to give the focus to the open slider and mark the visible items as "disabled" while the slider is open.
(similar to #4061)
![grafik](https://user-images.githubusercontent.com/1645099/160258032-1daf298e-cea9-4f03-8e46-4648261c8e3e.png)


Changes proposed in this pull request:

- CSS

How to test the feature manually:

1. open config -> extensions
2. edit an extension
3. see the greyed area left of the slider (that is the new "feature")
4. click on it
5. it closes the slider

Pull request checklist:
